backend/src/enums/user.type.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum UserTypeEnum {
+  NORMAL = 'normal',
```

Comment:
`INTERNAL = 'internal'` works better for me.

Reply:
agree on that